e2e/uimodel/bandwagon/bandwagon.go

```diff
@@ -0,0 +1,88 @@
+package bandwagon
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gravitational/robotest/e2e/framework"
+	"github.com/gravitational/robotest/e2e/uimodel/defaults"
+	"github.com/gravitational/robotest/e2e/uimodel/utils"
+
+	log "github.com/Sirupsen/logrus"
+	. "github.com/onsi/gomega"
+	web "github.com/sclevine/agouti"
+	. "github.com/sclevine/agouti/matchers"
+)
+
+// Bandwagon is bandwagon ui model
+type Bandwagon struct {
+	page *web.Page
+}
+
+// Open navigates to bandwagon URL and returns its ui model
+func Open(page *web.Page, domainName string) Bandwagon {
+	path := fmt.Sprintf("/web/site/%v", domainName)
+	url := utils.FormatUrl(page, path)
+	Expect(page.Navigate(url)).To(Succeed(), "should open bandwagon")
+	Eventually(page.FindByClass("my-page-btn-submit"), defaults.FindTimeout).
+		Should(BeFound(), "should wait for bandwagon to load")
+	return Bandwagon{page}
+}
+
+// SubmitForm submits bandwagon form
+func (b *Bandwagon) SubmitForm(config framework.BandwagonConfig) {
+	log.Infof("trying to submit bandwagon form")
+	if config.Organization == "" {
```

Comment:
not sure if it's a good approach to fill-in just some of the values with defaults. 
if no config is provided (config as a pointer == nil) then take a default one.